src/js/components/common/forms/formcheckbox.js

```diff
@@ -25,7 +25,9 @@ export const FormCheckbox = ({ className, control, id, handleClick, style, label
       return (
         <FormControlLabel
           className={className}
-          control={<Checkbox name={id} onClick={handleClick} checked={value} style={style} color="primary" onChange={() => onChange(!value)} />}
+          control={
+            <Checkbox {...checkboxProps} name={id} onClick={handleClick} checked={value} style={style} color="primary" onChange={() => onChange(!value)} />
```

Comment:
I would avoid props spreading as it damages readability (taking into account we don't use typescript)

Reply:
The spread aligns with [upstream/ mui](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Checkbox/Checkbox.js) though 😉 - should now be just the `disabled` prop 👌